rasa/core/actions/forms.py

```diff
@@ -524,7 +529,10 @@ async def _validate_if_required(
            - form validation was not cancelled
         """
         # no active_loop means that it is called during activation
```

Comment:
Is this comment still valid?

Reply:
I think so, if there's no active loop => form is not active => no events from form returned? @ancalita can you confirm?

Reply:
Maybe we can improve the comment: `no active_loop means there are no form filled slots to validate yet`?

Reply:
Yeah I was a bit confused as the behaviour changed but the comment stayed the same. I think Anca's suggestion of comment sounds good.

Reply:
Makes sense 👍